code/game/objects/structures/mineral_doors.dm

```diff
@@ -252,21 +252,19 @@
 	return
 
 /obj/structure/mineral_door/transparent/plasma/attackby(obj/item/W, mob/user, params)
-	if(W.is_hot())
-		var/turf/T = get_turf(src)
-		message_admins("Plasma mineral door ignited by [ADMIN_LOOKUPFLW(user)] in [ADMIN_VERBOSEJMP(T)]")
-		log_game("Plasma mineral door ignited by [key_name(user)] in [AREACOORD(T)]")
-		TemperatureAct()
+	if(W.is_hot() > 300)
```

Comment:
why the fuck does a proc called is_hot return a number

Reply:
not my proc, don't look at me.